cli/pkg/kctrl/cmd/app/init/github.go

```diff
@@ -118,3 +122,30 @@ func (g *GithubStep) initializeContentWithGithubRelease() error {
 	g.vendirConfig.Directories[0].Contents = append(g.vendirConfig.Directories[0].Contents, vendirconf.DirectoryContents{})
 	return g.initializeGithubRelease()
 }
+
+func (g *GithubStep) getIncludedPaths() error {
```

Comment:
Except for the small variance in the informational text, I don't see any difference in this and the getIncludedPaths for git. Perhaps this will be covered in the refactoring?

Reply:
After much discussion, we have decided to add this to the individual fetch option. Anyway, it will be covered in refactoring.